edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -3379,11 +3432,13 @@ QueryBubbleChartLiveReportController = Class.create(QueryLiveReportController, {
           1: this._userData[0][1],
           2: this._userData[0][2]
         });
-      }
+      };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1116)